certbot/docs/install.rst

```diff
@@ -124,26 +93,26 @@ of the ``/etc/letsencrypt`` directory, see :ref:`where-certs`.
 
 .. _Docker: https://docker.com
 .. _`install Docker`: https://docs.docker.com/engine/installation/
+.. _certbot.eff.org: https://certbot.eff.org/instructions
+
+Alternative 2: Pip
+------------------
+
+Installing Certbot through pip is only supported on a best effort basis and
+when using a virtual environment. Instructions for installing Certbot through
+pip can be found at https://certbot.eff.org/instructions by selecting your
+server software and then choosing "pip" in the "System" dropdown menu.
 
 .. _certbot-auto:
 
```

Comment:
Potentially we could list a third alternative here, which is distro packages.

It could be a good place to call out:

1. Their existence
2. That they are maintained by third-parties, unlike the methods listed here
3. That they tend to rapidly become out-of-date on LTS-style distros